openpower/patches/p8dtu-patches/linux/0001-VGA-handshake-with-BMC.patch

```diff
@@ -0,0 +1,68 @@
+From 4c5f6ff7a2b7c2df17d16c3ce18b19036be95223 Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Mon, 17 Dec 2018 15:57:05 -0800
+Subject: [PATCH] VGA handshake with BMC
```

Comment:
@shenki @amboar Are we any closer for a standard way to do this across openpower machines?

@shenki I'm gathering we don't particularly want to carry a platform specific kernel patch forever?

Reply:
We don't want to carry a platform specific patch ever.

Reply:
Yep, we have a standard way to do that, which doesn't require any host-side code. It's simply implemented by having the BMC watch for the host side setting the VGA password register after power-on. This means that the VGA driver is actively controlling the VGA device, and is all the signaling we should need.

@jimsmc if this design isn't sufficient, we'd need to know why before justifying a custom implementation of the VGA handshake.